utility/src/main/java/org/oppia/android/util/parser/BlurTransformation.kt

```diff
@@ -10,33 +10,53 @@ import com.bumptech.glide.load.engine.bitmap_recycle.BitmapPool
 import com.bumptech.glide.load.resource.bitmap.BitmapTransformation
 import java.security.MessageDigest
 
-/** [BlurTransformation] is a bitmap transformation that blurs an image using RenderScript. */
-class BlurTransformation(
-  private val context: Context,
-  private val radius: Float = 20f
-) : BitmapTransformation() {
+// The radius of the blur, a float between 0 and 25. It defines the value of the standard
+// deviation to the Gaussian function.
+private const val BLUR_RADIUS = 20f
+
+/**
+ * [BlurTransformation] is a bitmap transformation that blurs an image using RenderScript.
+ *
+ * The following [tutorial](https://futurestud.io/tutorials/glide-custom-transformation)
+ * was used as a reference, as well as this [article](https://stackoverflow.com/a/23119957).
+ * */
+
+class BlurTransformation(private val context: Context) : BitmapTransformation() {
```

Comment:
```suggestion
/**
 * [BlurTransformation] is a bitmap transformation that blurs an image using RenderScript.
 *
 * The following [tutorial](https://futurestud.io/tutorials/glide-custom-transformation)
 * was used as a reference, as well as this [article](https://stackoverflow.com/a/23119957).
 */
class BlurTransformation(private val context: Context) : BitmapTransformation() {
```

Minor stylistic fixes.